authorizer-app-backend/src/main/java/org/radarbase/authorizer/resources/RestSourceUserResource.kt

```diff
@@ -144,6 +143,23 @@ class RestSourceUserResource(
         return Response.noContent().header("user-removed", userId).build()
     }
 
+    @POST
+    @Path("revoke/sourceType/{sourceType}/externalId/{externalId}")
+    @NeedsPermission(Permission.Entity.MEASUREMENT, Permission.Operation.READ)
+    fun revokeToken(@PathParam("externalId") externalId: String,
+                    @PathParam("sourceType") sourceType: String,
+                    accessToken: TokenDTO): Boolean {
```

Comment:
Access token does not seem to be used. In any case, this service contains all access tokens, so it wouldn’t need the client to pass it.

Reply:
My bad the `revokeToken` method should have included the accessToken in the parameters. I was thinking that in the case where the user is not part of the repository, we can still revoke the token (using the accessToken as a request parameter) so that it won't keep sending data when it was not authorized (in the rest sources auth repo) in the first place?